requirements.txt

```diff
@@ -3,4 +3,5 @@ pulumi-gcp==6.49.0
 pulumi-azure-native==1.80.0
 pulumi-azuread==5.28.0
 toml==0.10.2
+xxhash==3.2.0
```

Comment:
Was this intentional?

Reply:
I included this and the below change as they made migrating the state easier.

This, so I can install directly with pip as it was missing. And the below change because it helped for Azure debugging, but then realised it wasn't all relevant anyway.

I can move these to a separate PR, but they do relate to the stack migration to infra private